packages/components/src/button/style.scss

```diff
@@ -154,12 +154,17 @@
 		white-space: nowrap;
 		color: var(--wp-admin-theme-color);
 		background: transparent;
-		padding: 6px; // This reduces the horizontal padding on tertiary/text buttons, so as to space them optically.
+		padding: $grid-unit-15 / 2; // This reduces the horizontal padding on tertiary/text buttons, so as to space them optically.
 
 		.dashicon {
 			display: inline-block;
 			flex: 0 0 auto;
 		}
+
+		// Pull left if the tertiary button stands alone after a description, so as to vertically align with items above.
```

Comment:
It seems strange that the styles of a button depend on if the button follows a paragraph or not. Is this change something that should happen all the time if before the button there is a paragraph?
For the other button styles would this change or something similar also make sense?
What if instead of adding these styles to the button we add them to the container of the description and button (widget areas block)?

Reply:
The thing is, the tertiary button has a built-in padding of 6px, which means it needs the similar negative margin only if it stands on its own. In the Image block example shown (primary, tertiary, tertiary) the button should not have negative margin.

Reply:
Thank you for the clarification 👍